frontend/ui/widget/opdsbrowser.lua

```diff
@@ -520,6 +525,13 @@ function OPDSBrowser:downloadFile(item, format, remote_url)
     end
 end
 
+function OPDSBrowser:createNewDownloadDialog(path, buttons)
+    self.download_dialog = ButtonDialogTitle:new{
+        title = _("Choose format to download into:") .. "\n" .. path,
```

Comment:
Heh, I didn't know if you had a problem with the length, so I suggested a more terse version. Yes, the "... into:" is meant to refer to the following path. So you're fine as is then?